gcs/src/gcs.cpp

```diff
@@ -380,7 +394,7 @@ gcs_create (gu_config_t* const conf, gcache_t* const gcache,
         goto recv_q_failed;
     }
 
-    conn->sm = gcs_sm_create(1<<16, 1);
+    conn->sm = gcs_sm_create(1<<16, 1, conn->params.fc_auto_evict_window);
```

Comment:
Please try to avoid implicit casts, "fc_auto_evict_window > 0" would make it clearer that this is a bool parameter.

Reply:
Fixed all in separate commit